src/pages/Homepage.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <SRNEntry />
+  <h3>OR</h3>
+  <StudentSelect />
+</template>
+
+<script>
+import StudentSelect from "../components/StudentSelect.vue";
+import SRNEntry from "../components/SRNEntry.vue";
```

Comment:
It's best to use absolute paths using `@`
`@` means the root directory which is the `src` directory

Because if we change the structuring of files, then this hardcoded path might break
```suggestion
import StudentSelect from "@/components/StudentSelect.vue";
import SRNEntry from "@/components/SRNEntry.vue";
```